### DISCUSSION TITLE
Has anyone used the Makerfabs 3.5in TFT touch with ESP32?

### DISCUSSION THREAD

**Discussion by tapd** [HASwitchPlate/openHASP#176] [Hardware]
I previously used the Nextion 2.4in screens with the original HASP project which worked well. Now I'm looking at the best option to test out openHASP and came across the Makerfabs display with in-built ESP32. It looks a lot like the Wireless-Tag display with a built in camera. https://www.makerfabs.com/esp32-3.5-inch-tft-touch-capacitive-with-camera.html
![image](https://user-images.githubusercontent.com/2671841/121483344-6a344f80-ca11-11eb-980f-1d3c16b604e5.png)
![image](https://user-images.githubusercontent.com/2671841/121483717-dadb6c00-ca11-11eb-8d51-1480bbe7e3ed.png)

Thank you all

**Comment by fvanroie** [comment]
They do seem a good fit for openHASP. I noticed these boards before but I don't have any... so support is not not verified yet. They have a couple of versions with 3.2" & 3.5" screen and capacitive or resistive touch. There's event an add-on board with DHT sensor, which is cool.

If you're able to compile the firmware yourself, we can probably get a working configuration... But it will be trial-and-error and no guarantees.

**Comment by fvanroie** [comment]
Support for the *ESP32 3.5" TFT Touch (Capacitive) with Camera* board will be [added in 0.6.1](https://haswitchplate.github.io/openHASP-docs/0.6.1/devices/makerfabs-tft-touch/)
